damus/Views/Profile/ProfileView.swift

```diff
@@ -162,7 +163,10 @@ struct ProfileView: View {
         Button(action: {
             action_sheet_presented = true
         }) {
-            navImage(img: "share3")
```

Comment:
This icon is misleading. It's generally understood to be "forward" or "share", but we have report and mute features in this menu as well.